spec/models/aaib_report_spec.rb

```diff
@@ -113,7 +94,7 @@ def aaib_report_content_item(n)
       expect(aaib_report.base_path).to            eq(aaib_reports[0]["base_path"])
       expect(aaib_report.title).to                eq(aaib_reports[0]["title"])
       expect(aaib_report.summary).to              eq(aaib_reports[0]["description"])
-      expect(aaib_report.body).to                 eq(aaib_reports[0]["details"]["body"])
+      expect(aaib_report.body).to                 eq(aaib_reports[0]["details"]["body"][0]["content"])
```

Comment:
Is the rendered HTML representation only added when the content is presented and sent downstream? In which case, this looks fine.